src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-form/nfs-form.component.html

```diff
@@ -419,6 +420,11 @@
                   *ngIf="nfsForm.getValue('access_type')">
               {{ getAccessTypeHelp(nfsForm.getValue('access_type')) }}
             </span>
+            <span class="form-text text-warning"
+                  *ngIf="nfsForm.getValue('access_type') === 'RW' && nfsForm.getValue('name') === 'RGW'">
+              <ng-container i18n>Object Gateway backend has some limitations regardings writes, please check the <a
+                   href="https://docs.ceph.com/docs/master/radosgw/nfs/"> documentation</a> for more information.</ng-container>
```

Comment:
Can we make this URL version-specific instead of pointing to the `master` documentation? To my knowledge we have other places in the dashboard where this functionality is used. Also, consider to make the warning a bit more explicit: 

> The Object Gateway NFS backend has a number of limitations which will seriously affect applications writing to the share. Please consult the documentation for details before enabling write access.

Reply:
Done